packages/aws-cdk-lib/aws-autoscaling/README.md

```diff
@@ -24,6 +24,7 @@ new autoscaling.AutoScalingGroup(this, 'ASG', {
 Creating an `AutoScalingGroup` from a Launch Configuration has been deprecated. All new accounts created after December 31, 2023 will no longer be able to create Launch Configurations. With the `@aws-cdk/aws-autoscaling:generateLaunchTemplateInsteadOfLaunchConfig` feature flag set to true, `AutoScalingGroup` properties used to create a Launch Configuration will now be used to create a `LaunchTemplate` using a [Launch Configuration to `LaunchTemplate` mapping](https://docs.aws.amazon.com/autoscaling/ec2/userguide/migrate-launch-configurations-with-cloudformation.html#launch-configuration-mapping-reference). Specifically, the following `AutoScalingGroup` properties will be used to generate a `LaunchTemplate`:
 * machineImage
 * keyName
```

Comment:
```suggestion
* keyName (deprecated, prefer keyPair)
```
Could you please add an example of `keyPair` usage in the code block below?

Reply:
Done in https://github.com/aws/aws-cdk/pull/29679/commits/b88ea014d76b60a0df1b17ca0a0469a839b3691b